torch/linalg/__init__.py

```diff
@@ -1684,15 +1705,17 @@
 
 Args:
     A (Tensor): tensor of shape `(*, m, n)` where `*` is zero or more batch dimensions.
-    rcond (float or Tensor, optional): the tolerance value to determine when is a singular value zero
-                                       If it is a :class:`torch.Tensor`, its shape must be
-                                       broadcastable to that of the singular values of
-                                       :attr:`A` as returned by :func:`torch.svd`.
-                                       Default: `1e-15`.
-    hermitian(bool, optional): indicates whether :attr:`A` is Hermitian if complex
-                               or symmetric if real. Default: `False`.
+    rcond (float, Tensor, optional): [NumPy compatibility] the tolerance value.
+                                     If the value of :attr:`rcond` is specified then
+                                     it's equivalent to setting `rtol=rcond`.
```

Comment:
What about the simpler:
```suggestion
    rcond (float, Tensor, optional): [NumPy Compat]. Alias for :attr:`rtol` . Default: `None`.
```

Also, should we put this here (the place where it semantically belongs), or at the end of `Keyword args`?

I am more inclined for the latter. This way people could find it in the docs, and we would still (silently) support NumPy programs, but we would announce this kind of args as little as possible. cc @mruberry